MFA API docs improvements (#9098)
auth0/docs: articles/mfa/concepts/mfa-api.md

useCase:
  - customize-mfa
---
# Multi-factor Authentication API

Auth0 provides a built-in <dfn data-key="multifactor-authentication">Multi-factor Authentication (MFA)</dfn> enrollment and authentication flow using [Universal Login](/universal-login).

You will need to use the MFA API in the following scenarios:

> - If you are authenticating users with the Resource Owner Password Grant. [Read more](/mfa/guides/mfa-api/authenticate).

Reviewer: ```suggestion
- If you are [authenticating users with the Resource Owner Password Grant](/mfa/guides/mfa-api/authenticate).
```

> - If you want to build an interface to let users manage their authentication factors. [Read More](/mfa/guides/mfa-api/manage).

Reviewer: ```suggestion
- If you want to build an interface to let users [manage their authentication factors](/mfa/guides/mfa-api/manage).
```

<%= include('../_includes/_authenticator-before-start') %>

## Limitations

> The MFA API is designed to work with SMS, Push via Guardian, Email, and OTP factors. It does not currently support enrolling with Duo or with the legacy 'google-authenticator' factor (which can be enrolled using the OTP factor).

Reviewer: ```suggestion
The MFA API is designed to work with SMS, Push via Guardian, Email, and OTP factors. It does not currently support enrolling with Duo or with the legacy 'google-authenticator' factor (Google Authenticator can still be enrolled using the OTP factor).
```
